build-info-extractor/src/test/java/org/jfrog/build/extractor/clientConfiguration/client/ArtifactoryBuildInfoClientTest.java

```diff
@@ -78,7 +78,7 @@ public void sendBuildInfoTest() throws IOException {
         buildInfoClient.sendBuildInfo(buildInfoToSend, null);
 
         // Get build info
-        Build receivedBuildInfo = buildInfoClient.getBuildInfo(BUILD_NAME, BUILD_NUMBER);
+        Build receivedBuildInfo = buildInfoClient.getBuildInfo(BUILD_NAME, BUILD_NUMBER, null);
```

Comment:
Let's enhance this test (or add another one) to test publishing build-info with project as well.
We may be able to reuse the entire code of this test, to test the usage of projects as well.

Reply:
Added test.